BAZProjectC1/View/Cells/MovieBasicInfoCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MovieBasicInfoCell.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 17/09/22.
+//
+
+import UIKit
+
+class MovieBasicInfoCell: UITableViewCell {
+
+    
+    @IBOutlet weak var movieTitle: UILabel!
+    @IBOutlet weak var moviePoster: UIImageView!
+    
+    static let  identificador = String(describing: MovieBasicInfoCell.self)
```

Comment:
Podríamos renombrar identificador a inglés? Para mantener consistencia en el idioma del código.
```suggestion
    static let identifier = String(describing: MovieBasicInfoCell.self)
```